config/shredder.batch.config.reference.hocon

```diff
@@ -68,6 +68,20 @@
     ]
   },
 
+  # Specifies interval shredder will work on
+  "runInterval": {
+    # Timestamp specifies since when shredder will process
+    "sinceTimestamp": "2021-10-12-14-55-22",
+    # Duration value specifies since when shredder will process.
```

Comment:
This is difficult field to explain!  I also had a go at it, see what you think:

> Optional, sinceAge is a duration that specifies the maximum age of folders that should get processed.  If sinceAge and sinceTimestamp are both specified, then the latest value of the two determines the earliest folder that will be processed.

Reply:
That is definitely much better! I will add it 👍